use-cases/prerequisites/storage-benchmarking.md

```diff
@@ -0,0 +1,129 @@
+# Storage Benchmarking Prerequisites
+
+These steps walk you through downloading Llama-3.3-70B-Instruct from Hugging
+Face into the two GCS buckets for use within the guide for the
+[storage benchmarking](/use-cases/inferencing/cost-optimization/storage-benchmarking/gcsfuse/).
+Please note that, you will be copying the model in two buckets, one of them is a
+flat bucket while the other is hierarchical.
+
+These prereqs were developed to be run on the
+[playground AI/ML platform](/platforms/gke-aiml/playground/README.md). If you
+are using a different environment the scripts and manifest will need to be
+modified for that environment.
+
+> NOTE: Due to the limitations of Cloud Shell’s storage and the size of llama
+> 70B model we need to run this transfer via Kubernetes job.
+
+- Get access to the model `meta-llama/Llama-3.3-70B-Instruct`
+
+  Go to https://huggingface.co/meta-llama/Llama-3.3-70B-Instruct . Sign In and
+  accept the license agreement to get the access to the model.
+
+- Change directory to the guide directory.
+
+  ```sh
+  cd use-cases/prerequisites
+  ```
+
+- Ensure that your `MLP_ENVIRONMENT_FILE` is configured
+
+  ```sh
+  cat ${MLP_ENVIRONMENT_FILE} && \
+  source ${MLP_ENVIRONMENT_FILE}
+  ```
+
+  > You should see the various variables populated with the information specific
+  > to your environment.
+
+- Get credentials for the GKE cluster
+
+  ```sh
+  gcloud container fleet memberships get-credentials ${MLP_CLUSTER_NAME} --project ${MLP_PROJECT_ID}
+  ```
+
+- Set `HF_TOKEN` to your HuggingFace access token. Go to
+  <https://huggingface.co/settings/tokens> , click `Create new token` , provide
+  a token name, select `Read` in token type and click `Create token`.
+
+  ```sh
+  HF_TOKEN=
+  ```
+
+- Set accelerator
+
+  ```sh
+  export ACCELERATOR=a100
+  ```
+
+- Create a Kubernetes secret to hold the Hugging Face token
+
+  ```sh
+  kubectl create secret generic hf-secret \
+    --from-literal=HF_TOKEN=${HF_TOKEN} \
+    --dry-run=client -o yaml | \
+    kubectl --namespace ${MLP_KUBERNETES_NAMESPACE} apply  -f -
+  ```
+
+- Replace the respective variables required for the job
+
+  ```sh
+  MODEL_REPO=meta-llama/Llama-3.3-70B-Instruct
+
+  sed \
+    -i -e "s|V_KSA|${MLP_STORAGE_BENCHMARKING_KSA}|" \
+    -i -e "s|V_FLAT_BUCKET|${MLP_STORAGE_BENCHMARK_FLAT_BUCKET}|" \
+    -i -e "s|V_HIERARCHICAL_BUCKET|${MLP_STORAGE_BENCHMARK_HIERARCHICAL_BUCKET}|" \
+    -i -e "s|V_MODEL_REPO|${MODEL_REPO}|" \
+    manifests/transfer-llama-to-gcs-${ACCELERATOR}-dws.yaml
```

Comment:
Switch this to use `envsubst` similar to the [RAG guides](https://github.com/GoogleCloudPlatform/accelerated-platforms/blob/main/use-cases/rag-pipeline/embedding-models/multimodal-embedding/README.md)